packages/manager/modules/pci/src/projects/project/kubernetes/service.js

```diff
@@ -189,4 +196,55 @@ export default class Kubernetes {
   resetInstancesCache() {
     this.OvhApiCloudProjectInstance.v6().resetCache();
   }
+
+  getPrivateNetworks(projectId) {
+    return this.$http
+      .get(`/cloud/project/${projectId}/network/private`)
+      .then((networks) =>
+        filter(networks.data, {
```

Comment:
Can't filter when fetching ?

Reply:
I think you mean by using iceberg.
I think we can but because tomorrow we plan to prod this, it's more safe to not change this.

Thanks.